src/main/resources/db/migration/V006__Add_status_column.sql

```diff
@@ -1,2 +1,14 @@
 ALTER TABLE letters
 ADD COLUMN status VARCHAR(256);
+
+UPDATE
+  letters
+SET
+  status = (
+    CASE
+      WHEN printed_at IS NULL THEN
+        'Uploaded'
+      ELSE
+        'Posted'
+    END
+  );
```

Comment:
parhaps we could make it `NOT NULL` after that?

Reply:
done